cmd/kubeadm/app/util/apiclient/idempotency.go

```diff
@@ -60,13 +60,22 @@ func CreateOrUpdateConfigMap(client clientset.Interface, cm *v1.ConfigMap) error
 // to conflicts, and a retry will be issued if the ConfigMap was modified on the server between the refresh and the update (while the mutation was
 // taking place)
 func CreateOrMutateConfigMap(client clientset.Interface, cm *v1.ConfigMap, mutator ConfigMapMutator) error {
-	if _, err := client.CoreV1().ConfigMaps(cm.ObjectMeta.Namespace).Create(cm); err != nil {
-		if !apierrors.IsAlreadyExists(err) {
+	return clientsetretry.OnError(wait.Backoff{
```

Comment:
give the retry condition is always true, we should just use retry.ExponentialBackoff() here.
similar to https://github.com/kubernetes/kubernetes/blob/6049ec28b1a8b88dc5ebe09d72e890936ce707fc/cmd/kubeadm/app/util/apiclient/idempotency.go#L325
with lastError storage.